machine_manager/README.rst

```diff
@@ -0,0 +1,27 @@
+Machine Manager
```

Comment:
@Daniel-CA @anajuaristi 
When somebody will move this code http://bazaar.launchpad.net/~ingenieria-adhoc/adhoc-oerp/6.1/view/head:/network/network.py to v8, then he will contribute on the base equipment module comes from your code. 
Then your maintenance work on each new Odoo version will be shared with him. It's a good news for your timeline job. 
Yes there is a little job to split your module but it's a really good news for a long time support.
Thanks you to consider it.